pkg/maps/eppolicymap/eppolicymap.go

```diff
@@ -42,88 +42,10 @@ var (
 	EpPolicyMap *bpf.Map
 )
 
-// CreateWithName creates a new endpoint policy hash of maps for
-// looking up an endpoint's policy map by the endpoint key.
-//
-// The specified mapName allows non-standard map paths to be used, for instance
-// for testing purposes.
-func CreateWithName(mapName string) error {
```

Comment:
Thanks, but I think the whole `eppolicy` package is now empty and can be removed.